proof/proof_stitching_test.go

```diff
@@ -121,9 +127,7 @@ func TestProofStitching(t *testing.T) {
 		// get an "invalid exclusion proof" error when validating, we
 		// need to skip this proof.
 		_, err = f.Verify(
-			context.Background(), verifier.VerifyHeader,
-			DefaultMerkleVerifier, MockGroupVerifier,
-			MockChainLookup,
+			context.Background(), vCtx,
```

Comment:
Here we actually switch to the `DefaultMerkleVerifier` instead of the `MockMerkleVerifier`, since we now have a valid merkle proof (which IIRC is part of the test case).